src/patterns/uac/index.njk

```diff
@@ -1,5 +1,5 @@
 ---
-title: Unique Access Codes
+title: Access Codes
 group: Ask users for...
 ---
 {% from "views/partials/example/_macro.njk" import patternlibExample %}
```

Comment:
Add a 'This pattern requires documentation.' panel under patter page heading, until we can document this properly